src/components/Kurlsh.js

```diff
@@ -926,34 +936,39 @@ class Kurlsh extends React.Component {
 
               <div className="flex flex-column u-marginTop--40">
                 <span className="u-fontSize--normal u-fontWeight--medium u-color--bermudaGray"> CNI plugin </span>
-                {/* hiding calico add on */}
-                <div className={`u-display--none AddOn--wrapper ${selectedVersions.calico.version !== "None" && "selected"} u-marginTop--15`} onClick={(e) => this.handleIsAddOnSelected("calico", e)}>
+                <div className={`AddOn--wrapper ${selectedVersions.antrea.version !== "None" && "selected"} flex flex-column u-marginTop--15`} onClick={(e) => this.handleIsAddOnSelected("antrea", e)}>
                   <div className="flex flex1">
                     <div className="flex flex1 alignItems--center">
                       <input
                         type="checkbox"
                         className="u-marginRight--normal"
-                        checked={selectedVersions.calico.version !== "None"}
+                        checked={selectedVersions.antrea.version !== "None"}
                       />
-                      <span className="icon u-calico u-marginBottom--small" />
+                      <span className="icon u-velero u-marginBottom--small" />
```

Comment:
Velero is the closest icon to Antrea. Need to figure out how to add the antrea icon to the spritesheet, then change this to `u-antrea`.